specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/extensions.json

```diff
@@ -310,45 +443,83 @@
         },
         "primaryKey": {
-          "description": "The certificate for the cluster monitoring extensions.",
+          "description": "The certificate for the Azure Monitor Classic extensions.",
           "type": "string"
         }
       }
     },
-    "ClusterMonitoringResponse": {
-      "description": "The Operations Management Suite (OMS) status response",
+    "AzureMonitorClassicResponse": {
+      "description": "The Azure Monitor Classic (OMS) status response",
       "properties": {
         "clusterMonitoringEnabled": {
-          "description": "The status of the Operations Management Suite (OMS) on the HDInsight cluster.",
+          "description": "The status of the Azure Monitor Classic (OMS) on the HDInsight cluster.",
           "type": "boolean"
         },
         "workspaceId": {
-          "description": "The workspace ID of the Operations Management Suite (OMS) on the HDInsight cluster.",
+          "description": "The workspace ID of the Azure Monitor Classic (OMS) on the HDInsight cluster.",
           "type": "string"
         }
       }
     },
-    "ClusterMonitoringRequest": {
-      "description": "The Operations Management Suite (OMS) parameters.",
+    "AzureMonitorClassicRequest": {
+      "description": "The Azure Monitor Classic (OMS) parameters.",
       "properties": {
         "workspaceId": {
-          "description": "The Operations Management Suite (OMS) workspace ID.",
+          "description": "The Azure Monitor Classic (OMS) workspace ID.",
           "type": "string"
         },
         "primaryKey": {
-          "description": "The Operations Management Suite (OMS) workspace key.",
+          "description": "The Azure Monitor Classic(OMS) workspace key.",
           "type": "string"
         }
       }
-    }
-  },
+    },
+	"description": "Azure Monitor extensions",
+      "properties": {
+        "workspaceId": {
+          "description": "The workspace ID for the Azure Monitor Integration extension.",
+          "type": "string"
+        },
+        "primaryKey": {
+          "description": "The workspace key for the Azure Monitor Integration extensions.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureMonitorResponse": {
+      "description": "The Azure Monitor Integration status response",
+      "properties": {
+        "clusterMonitoringEnabled": {
+          "description": "The status of the Azure Monitor Integration on the HDInsight cluster.",
+          "type": "boolean"
+        },
+        "workspaceId": {
+          "description": "The workspace ID of the Azure Monitor Integration on the HDInsight cluster.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureMonitorRequest": {
+      "description": "The Azure Monitor Integration request parameters.",
+      "properties": {
+        "workspaceId": {
+          "description": "The Azure Monitor Integration  workspace ID.",
+          "type": "string"
+        },
+        "primaryKey": {
```

Comment:
You said that you plan to add a new optional parameter to enable multiple put request. But from this definition, I can find the optional parameter besides the already existing parameters "workspaceId" and "primaryKey".